code/modules/mob/living/carbon/human/human.dm

```diff
@@ -894,9 +894,14 @@
 	if(istype(location, /turf))
 		location.add_vomit_floor(src, 1)
 
+	if(reagents.reagent_list.len)
```

Comment:
length(), not .len

Reply:
if(reagents.reagent_list.length())

Reply:
Sorry, 	if(length(reagents.reagent_list))

My javascript background comes back to haunt me.